Automattic/wp-calypso
# Create data layer for Marketplace reviews

Related to https://github.com/Automattic/dotcom-forge/issues/4556

## Proposed Changes

* add methods to use the new review API endpoints
* add some elements to the marketplace test site to be able to test requests

## Testing Instructions



* Apply the patch in D129780-code and sandbox `public-api`
* Go to `/marketplace/test/:siteID`
* Use the new card on the top of the page to add/update/delete reviews
* Check that the query on the top of the page is refetched after each operation

![CleanShot 2023-11-22 at 13 05 08@2x](https://github.com/Automattic/wp-calypso/assets/11555574/1e77bf66-a1f9-49e4-a02a-a701499237c1)

 
## Pre-merge Checklist
